tests/unit/internal-session-test.js

```diff
@@ -215,18 +253,23 @@ describe('InternalSession', () => {
         });
       });
 
-      it('persists its content in the store', () => {
-        return session.authenticate('authenticator').then(() => {
-          let properties = store.restore();
-          delete properties.authenticator;
+      it('persists its content in the store', (done) => {
```

Comment:
You can simply return a promise from the test instead of using `done` - I think that's a bit cleaner and also consistent with the rest of the code base then.